packages/ckeditor5-clipboard/docs/_snippets/features/build-paste-source.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @license Copyright (c) 2003-2020, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/* globals window */
+
+import ClassicEditor from '@ckeditor/ckeditor5-build-classic/src/ckeditor';
```

Comment:
The build setup could use some adjustment here, @Mgsy can you help? The plugins imported do not really correspond to what we need in the sample.

Reply:
I've reduced the amount of plugins and left only ones related to formatting.